index.ps1

```diff
@@ -116,6 +86,9 @@ if ($MyInvocation.Line.Trim() -notmatch '^\.\s+') {
         } else {
             $r = processRequest $request
 
+            if ($r.ContainsKey('stacktrace')) {
+                $response.StatusCode=500
```

Comment:
Is it possible that the function could return a valid result that contains `stacktrace`? We wouldn't want to return a 500 unless an exception occured.